docs/UserGuide.md

```diff
@@ -360,6 +360,25 @@ Format: `delete_session n/STUDENT_NAME i/SESSION_INDEX`
 Example:
 * `delete_session n/John Lee i/1` deletes John Lee's **first** session
 
+#### Deleting a recurring tuition session: `delete_rec_session`
+
+Deletes a single session from an existing recurring tuition session of a specified student from TutorBuddy.
+
+Format: `delete_rec_session n/STUDENT_NAME i/SESSION_INDEX d/DATE t/TIME`
+
+Arguments similar to `delete_session` command except the following:
+* `DATE` should be the date of the single session to be deleted
+* `TIME` should be the time of the single session to be deleted
```

Comment:
Actually not related to the documentation, but why do we need time here again ah? Cos once we have the student name and the session index, we would have known the time already right? There can only be 1 time right.

Reply:
Is the removal of t/TIME gonna be done by 1.3 or is it a 1.4 thing?

Reply:
Think it would be safer to postpone this to 1.4, because I'll need to abstract away from `SessionDate` or use a dummy time in `SessionDate`

Reply:
Ok sure! Got it!

Reply:
Updated #158 for this. Thanks all.